queries.sql

```diff
@@ -1,10 +1,59 @@
-/*Queries that provide answers to the questions from all projects.*/
-
-SELECT * FROM animals WHERE name LIKE '%mon';
-SELECT * FROM animals WHERE date_of_birth BETWEEN '1-1-2016' AND '31-12-2019';
-SELECT * FROM animals WHERE neutered = '1' AND escape_attempts < 3;
-SELECT date_of_birth FROM animals WHERE name IN ('Agumon', 'Pikachu');
-SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
-SELECT * FROM animals WHERE neutered = '1';
-SELECT * FROM animals WHERE name NOT LIKE 'Gabumon';
-SELECT * FROM animals WHERE weight_kg >= 10.4 AND weight_kg <= 17.3;
+BEGIN;
+-- Start a Transaction:
+BEGIN
+
+-- Update "species" to "unspecified" for Animals with Names Ending in "mon":
+UPDATE
+  animals
+SET
+  species = 'unspecified'
+WHERE
+  name LIKE '%mon';
+
+UPDATE
+  6
+
+  -- Update "species" to "pokemon" for Animals without a Species:
+UPDATE
+  animals
+SET
+  species = 'pokemon'
+WHERE
+  species IS NULL;
+
+UPDATE
+  5 -- TAKE A BREATH 
+  BEGIN;
+
+BEGIN

```

Comment:
- [ ] In this section, the requiremnet is to:
- Update the animals table by setting the species column to digimon for all animals that have a name ending in mon.
- Update the animals table by setting the species column to pokemon for all animals that don't have species already set.
- Verify that changes were made.
- Commit the transaction.
- Verify that changes persist after commit.

- [ ] It turns out you missed the change from digimon to mon, the commit, and verification part of this section. Kindly carry it out based on the requirements. The idea is to make sure you verify each change as it goes 👍 

### Requirements:
![image](https://github.com/Mike47ip/vet-clinic-database/assets/50990925/6a0f6456-5f93-405c-9f73-5c877fb6f546)

### Sample code:
```
   BEGIN;
   UPDATE animals SET species = 'digimon' WHERE name LIKE '%mon'; 
   UPDATE animals SET species = 'pokemon' WHERE species IS NULL; 

   SELECT species from animals; -- verify that change was made
   COMMIT;
   SELECT species from animals; -- verify that change persists after commit
```